zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -252,6 +255,10 @@ object Server {
       reqHandler      = settings.app.compile(zExec, settings, ServerTime.make)
       init            = ServerChannelInitializer(zExec, settings, reqHandler)
       serverBootstrap = new ServerBootstrap().channelFactory(channelFactory).group(eventLoopGroup)
+      _               = if (settings.option.nonEmpty)
+        settings.option.foldLeft[ServerBootstrap](serverBootstrap)((serverBootstrap, i) =>
+          serverBootstrap.option(i.channelOption, i.value),
+        )
```

Comment:
The feature is requesting a way to set both `option` and `childOption` on ServerBootstrap. Looks like it only sets option.